device/simulation/tt_simulation_device.h

```diff
@@ -57,24 +55,20 @@ class tt_SimulationDevice: public tt_device {
     // Misc. Functions to Query/Set Device State
     // virtual bool using_harvested_soc_descriptors();
     virtual std::unordered_map<chip_id_t, uint32_t> get_harvesting_masks_for_soc_descriptors();
-    // virtual bool noc_translation_en();
     // virtual void translate_to_noc_table_coords(chip_id_t device_id, std::size_t &r, std::size_t &c);
```

Comment:
Why are these functions under comment? Are these functions used anywhere?

Reply:
Ah no they're not used. Originally copy pasted from `tt_silicon_driver` and then commented out functions that I didn't need for the simulation but then forgot about them

I'll delete these commens